Updated README file

prakashpp opened the pull request.

Commit:
Updated README file

rituparnapanda commented on README.rst:
```diff
@@ -1 +1,46 @@
+
+.. image:: images/payroll.png
+
+Now go to ``Import Payroll Items wizard``, choose Credit Account,
```
Now go to `Import Payroll Items wizard` (Financial >> Entries >> Import Payroll Items)

rituparnapanda commented on README.rst:
```diff
@@ -1 +1,46 @@
+.. image:: images/payroll.png
+
+Now go to ``Import Payroll Items wizard``, choose Credit Account,
+account which will have effective balance and csv file of quickbooks payroll entries.
```
and click the search icon to select the csv file of quickbooks payroll entries from your computer.  Now click `Import`

Commit:
Fixed tests

Previously the data for test import had a date which is out of the
period now. Hence wrote a dynamic csv file for test.
